index.js

```diff
@@ -82,14 +82,15 @@ module.exports = function({parent}) {
                 'headers': headers,
                 'body': msg.payload
             };
-            // if there is a raw config property it will be merged with `connProps`
+            // if there is a raw config property it will be merged with `reqProps`
             if (this.config.raw) {
-                Object.assign(connProps, this.config.raw);
+                Object.assign(reqProps, this.config.raw);
             }
 
             // do the connection + request
-            let req = request(connProps, (error, response, body) => {
+            let req = request(reqProps, (error, response, body) => {
                 if (error) { // return error if any
+                    error.request = reqProps;
```

Comment:
This is potential security issue as sensitive data can leak to logs or even to the invoking front-end. Maybe only enable it if debug=true